plugins/PostProcessingPlugin/scripts/TimeLapse.py

```diff
@@ -0,0 +1,100 @@
+# Created by Wayne Porter
+
+from ..Script import Script
+
+class TimeLapse(Script):
+    def __init__(self):
+        super().__init__()
+
+    def getSettingDataString(self):
+        return """{
+            "name": "Time Lapse",
+            "key": "TimeLapse",
+            "metadata": {},
+            "version": 2,
+            "settings":
+            {
+                "trigger_cmd":
```

Comment:
Do not abbreviate setting name:
```suggestion
                "trigger_command":
```